az-digital/az_quickstart
# Closes #1103: Add new AZ News view displays

## Description

Adds new view displays to AZ News view
- Feature: block with 1 news article sorted by sticky first, then publication date
- Grid with filter: block with 9 news articles sorted by publication date, filterable by category

## Related issues




Closes #1103 
**BLOCKED BY #1500** 

## How to test


1. Create a news article with older publication date
2. Make sticky at top of lists (right column)
3. Create a new page
4. Add view > AZ News > Feature
5. Verify sticky news article appears
6. Add view to page > AZ News > Grid with Filter
7. Verify filter works
8. Change items per page in view settings on page
9. Verify items per page works
10. Verify pager works
11. Add other AZ News views 
12. Verify no inadvertent changes to other AZ News views

### Note:
_These are **block** displays. No page at a searchable path will be made. These views are not attached/connected to each other._ 
#### Reasoning
- Adding another page to the site in addition to /news and /news-teasers starts to create bloat and confusion for end-users
- Two views can be added to a page and better managed by content editors
- Separate blocks provides more flexibility - can use one or the other or both

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- Patch release changes
- Minor release changes

## Checklist


- [x] My change requires a change to the documentation. (QS site)
